src/constants/routes.js

```diff
@@ -0,0 +1,15 @@
+import Home from '../pages/home';
+import About from '../pages/about';
+import Contact from '../pages/contact';
+import Criteria from '../pages/criteria';
+import Promote from '../pages/promote';
+
+const routes = [
+  { path: '/', component: Home, exact: true },
+  { path: '/about', component: About },
+  { path: '/criteria', component: Criteria },
+  { path: '/promote your business', component: Promote },
```

Comment:
Instead of spaces, can we do - (or something that isn't spaces)? I believe these routes will reflect on the URL of the website and I believe URLs can't have spaces in theme